src/java/io/bazel/rulesscala/worker/BUILD

```diff
@@ -1,12 +1,7 @@
-load("@rules_java//java:defs.bzl", "java_library")
-
 java_library(
     name = "worker",
-    srcs = [
-        "GenericWorker.java",
-        "Processor.java",
-    ],
-    visibility = ["//visibility:public"],
+    srcs = ["Worker.java"],
+    visibility = ["//:__subpackages__"],
```

Comment:
We actually depend on this target for a scalapb worker. I think it's no longer needed since we moved to aspects but not 100% sure.
Sounds like this has value though maybe it should be in rules_java or somewhere lese.
WDYT?

Reply:
Is sounds like you have two suggestions here, both of which are reasonable to me:
1. Keep the worker base public and consider it part of our public API for others to use.
2. Move the worker base out to another repo so other folks can more easily depend on it.

I'd do 1 now, then move to 2 if the need is clear.